int8 planning_mode
float64 allocated_planning_time

# added for full body experiments without gazebo
float64[] swarm_start
float64[] swarm_goal

# tolerance
float64 tolerance

float64 initial_eps
float64 final_eps
float64 dec_eps

int8 planner_type
int8 meta_search_type

---

# planning stats
string[] stats_field_names
float64[] stats
